tidy.go

```diff
@@ -11,7 +11,7 @@ import (
 // Errors & warnings are deliberately suppressed as underlying tools
 // throw warnings very easily.
 func Tidy(r io.Reader, xmlIn bool) ([]byte, error) {
-	f, err := ioutil.TempFile("/tmp", "sajari-convert-")
+	f, err := ioutil.TempFile(os.TempDir(), "sajari-convert-")
```

Comment:
Is there a reason this doesn't use `NewLocalFile`

Reply:
no reason i can see

Reply:
```suggestion
	f, err := NewLocalFile(r)
```